c2corg_ui/templates/outing/helpers/view.html

```diff
@@ -292,33 +292,33 @@ <h3 ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-ta
 </%def>
 
 ## RATING
-<%def name="get_outing_route_rating(route)">\
-% if route.get('global_rating') or route.get('engagement_rating') or route.get('risk_rating') or route.get('exposition_rock_rating') \
-  or route.get('rock_free_rating') or route.get('rock_required_rating') or route.get('hiking_rating') \
-  or route.get('hiking_mtb_exposition') or route.get('ice_rating') or route.get('mixed_rating') or route.get('snowshoe_rating')  \
-  or route.get('mtb_down_rating') or route.get('mtb_up_rating') or route.get('via_ferrata_rating') or route.get('equipment_rating') \
-  or route.get('ski_rating') or route.get('labande_ski_rating') or route.get('labande_global_rating') or route.get('ski_exposition') :
+<%def name="get_outing_rating(ouring)">\
```

Comment:
As mentioned by @desnoes it should be "outing" instead of "ouring".

Reply:
Please note that all those attributes are not available in outing documents. The list of available attributes is here: https://github.com/c2corg/v6_api/issues/685#issuecomment-333570449